frontend/src/components/navbar/Navbar.js

```diff
@@ -3,12 +3,13 @@ import Logo from "../../assets/Logo.jpeg";
 import CommonButton from "../button/commonButton/CommonButton";
 import { useAuth } from "../../contexts/UserContext";
 
-import React, { useEffect } from "react";
-import Axios from "axios";
+import React, { useEffect, useState } from "react";
 import { Link, useNavigate } from "react-router-dom";
 
 function Navbar() {
-  const { currentUser, setCurrentUser } = useAuth();
+  const { setCurrentUser } = useAuth();
+  var [isLoggedIn, setIsLoogedIn] = useState(false);
```

Comment:
why `var`?

Reply:
As my isLoggedIn value has to change  upon logout

Reply:
so what? It is a state and it can change.